infer_structcol/tests/test_model.py

```diff
@@ -24,21 +24,28 @@ def test_calc_resid_spect():
     assert_frame_equal(calc_resid_spect(spect2, spect1), expected_output)
 
 def test_log_prior():
```

Comment:
Agreed, but will leave this for a future pull request since it is not a priority at this moment.